code/obj/item/device/scanners.dm

```diff
@@ -522,14 +522,14 @@ that cannot be itched
 ///////////////////////////////////////////////// Prisoner scanner ////////////////////////////////////
 
 /obj/item/device/prisoner_scanner
-	name = "Securotron-5000"
-	desc = "Used to scan in prisoners and update their security records."
-	icon_state = "forensic0"
+	name = "Security RecordTrak"
+	desc = "A device used to scan in prisoners and update their security records."
+	icon_state = "securotron"
 	var/mode = 1
 	var/datum/data/record/active1 = null
 	var/datum/data/record/active2 = null
 	w_class = 3.0
-	item_state = "electronic"
+	item_state = "securotron"
```

Comment:
If you're renaming the thing, don't rename the item_state to the old name. Give the item_state something like the new name.